joss/paper.md

```diff
@@ -0,0 +1,108 @@
+---
+title: 'signalgp-lite: A genetic programming implementation designed for large-scale artificial life applications'
+tags:
+  - C++
+  - artificial life
+  - genetic programming
+  - event driven
+authors:
+  - name: Matthew Andres Moreno
+    orcid: 0000-0003-4726-4479
+    affiliation: 1
+  - name: Santiago Rodriguez Papa
+    orcid: 0000-0002-6028-2105
+    affiliation: 1
+affiliations:
+ - name: Michigan State University
+   index: 1
+date: 31 May 2021
+bibliography: paper.bib
+---
+
+# Abstract
+
+Event-driven genetic programming has been shown to outperform traditional methods of
+
+# Statement of need
+
+`Gala` is an Astropy-affiliated Python package for galactic dynamics. Python
+enables wrapping low-level languages (e.g., C) for speed without losing
+flexibility or ease-of-use in the user-interface. The API for `Gala` was
+designed to provide a class-based and user-friendly interface to fast (C or
+Cython-optimized) implementations of common operations such as gravitational
+potential and force evaluation, orbit integration, dynamical transformations,
+and chaos indicators for nonlinear dynamics. `Gala` also relies heavily on and
+interfaces well with the implementations of physical units and astronomical
+coordinate systems in the `Astropy` package [@astropy] (`astropy.units` and
+`astropy.coordinates`).
+
+`Gala` was designed to be used by both astronomical researchers and by
+students in courses on gravitational dynamics or astronomy. It has already been
+used in a number of scientific publications [@Pearson:2017] and has also been
+used in graduate courses on Galactic dynamics to, e.g., provide interactive
+visualizations of textbook material [@Binney:2008]. The combination of speed,
+design, and support for Astropy functionality in `Gala` will enable exciting
+scientific explorations of forthcoming data releases from the *Gaia* mission
+[@gaia] by students and experts alike.
+
+# Mathematics
+
+Single dollars ($) are required for inline mathematics e.g. $f(x) = e^{\pi/x}$
+
+Double dollars make self-standing equations:
+
+$$\Theta(x) = \left\{\begin{array}{l}
+0\textrm{ if } x < 0\cr
+1\textrm{ else}
+\end{array}\right.$$
+
+You can also use plain \LaTeX for equations
+\begin{equation}\label{eq:fourier}
+\hat f(\omega) = \int_{-\infty}^{\infty} f(x) e^{i\omega x} dx
+\end{equation}
+and refer to \autoref{eq:fourier} from text.
+
+# Results
+
```

Comment:
We should add a few sentences to the beginning of the results explaining that we did the replications to "integration" test the functionality of our implementation. They also serve as reference demos for new users.